app/addons/documents/queryoptions/queryoptions.react.jsx

```diff
@@ -343,6 +347,10 @@ A key value is the first parameter emitted in a map function. For example emit("
       Actions.toggleQueryBarVisibility(!this.props.contentVisible);
     },
 
+    closeTray: function () {
+      Actions.toggleQueryBarVisibility(false);
```

Comment:
question: there are a few other occurences, like https://github.com/apache/couchdb-fauxton/pull/664/files#diff-a9b641a64002ae88871f5b368a99f1beL351 -- i think we have to replace them, too in order to make the callbacks called uniform?